bindings/python/src/datetime.cpp

```diff
@@ -55,6 +56,33 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point tpt)
+    {
+        object result;
+        if (tpt > lt::min_time()) {
+            lt::time_point now = lt::clock_type::now();
+            time_t const tm = lt::total_seconds(tpt - now);
```

Comment:
This is a refactor of your earlier suggested code, I did wonder that before but I couldn't get a valid `tpt` value so wasn't able to verify